files/galaxy-test/dynamic_rules/usegalaxy/sorting_hat.py

```diff
@@ -161,6 +170,17 @@ def build_spec(tool_spec, runner_hint=None):
         if 'rank' in tool_spec:
             params['rank'] = tool_spec['rank']
 
+    if 'remote_condor_cluster_mq' in destination:
```

Comment:
the memory usage does not need to be set explicitly?

Reply:
is done in the kwargs block